plugin.video.dailymotion_com/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.2.3" provider-name="L0RE">
+<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.3.3" provider-name="gujal">
```

Comment:
You have to increase version to a higher one then `2.3.3`, otherwise it won't get regenerated.

Reply:
Changed as per your advice
Thanks